actions/gha-workflow-validator/jest.config.ts

```diff
@@ -0,0 +1,13 @@
+import {Config} from 'jest'
+export default {
+  displayName: "gha-workflow-validator",
+  preset: "../../jest.preset.js",
+  testEnvironment: "node",
+  transform: {
+    "^.+\\.[tj]s$": "@swc/jest",
```

Comment:
```suggestion
    "^.+\\.ts$": "@swc/jest",
```
Do we need to do any js transforms?

Reply:
Seems like it's best to include if we have to include js for `moduleFileExtensions`?